src/SDSS.jl

```diff
@@ -130,5 +135,454 @@ function load_stamp_catalog(cat_dir, stamp_id, blob; match_blob=false)
 end
 
 
+@doc """
+Load data from a psField file, which contains the point spread function.
+
+Args:
+ - field_dir: The directory of the file
+ - run_num: The run number
+ - camcol_num: The camcol number
+ - frame_num: The frame number
+ - b: The filter band (a number from 1 to 5)
```

Comment:
Maybe better to call b "The filter band id (a number from 1 to 5)", so as not to confuse it with the letter, or the band itself